tests/integration/cloud/clouds/test_hcloud.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""
+    Tests for the Hetzner-Public-Cloud driver
```

Comment:
@dithmer i just found that here is still refered to "Public-Cloud" (and below too)